common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -0,0 +1,220 @@
+#### Local Types ####
```

Comment:
Works better as the hive breeding art IMHO.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105626477-4d73f280-5e49-11eb-844a-d21181cd07be.png)

</details>